src/index.js

```
@@ -3,7 +3,16 @@
 'use strict';
 
 const { createServer } = require('./createServer');
+const { sequelize } = require('./db');
 
-createServer().listen(5700, () => {
-  console.log('Server is running on localhost:5700');
-});
+async function start() {
+  try {
+    await sequelize.sync();
+    console.log('✅ Database synchronized');
+
+    createServer().listen(5700, () => {
+      console.log('Server is running on localhost:5700');
+    });
+  } catch {}
```

The catch block is empty which silently swallows any error thrown during `sequelize.sync()` or server startup. This can hide failures when tests or the app try to sync the DB (violating the requirement that models be reliably synchronized). Consider at minimum logging the error (e.g., `console.error(error)`) and exiting with a non-zero code so problems are visible and tests don't silently pass/fail unexpectedly. Example guidance: handle the error and `process.exit(1)` or rethrow after logging.